src/plugins/auto_batch/tests/unit/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@
 
 set(TARGET_NAME ov_auto_batch_unit_tests)
 
+if(SUGGEST_OVERRIDE_SUPPORTED)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-suggest-override")
+endif()
```

Comment:
Why did you add this flag to auto batch unit tests?

Reply:
I faced with this type of errors when I was making this changes. Now I checked and this flag is redundant.